src/main/webapp/stats.html

```diff
@@ -1,41 +1,50 @@
-<!DOCTYPE html>

```

Comment:
I'm confused about why this appears to be showing the entire file changing instead of just showing the changes you have made. Are you making changes on 2 different computers that are using different line endings? Please make sure every machine you are working on use unix style line endings and not DOS style.